test/fixtures/export/Do not caps-protect name fields #384.bibtex

```diff
@@ -34,12 +34,12 @@
 }
 
 @book{bovendeert,
-  author = {{boven d'\relax{} Eert}, Christianus},
+  author = {{boven d'Eert}, Christianus},
```

Comment:
I don't understand... I thought you said NDPs need a `\relax ` and a space for BibTeX?